plenum/server/primary_selector.py

```diff
@@ -89,3 +92,60 @@ def next_primary_replica_name_for_backup(self, instance_id, master_primary_rank,
     def start_election_for_instance(self, instance_id):
         raise NotImplementedError("Election can be started for "
                                   "all instances only")
+
+    def on_catchup_complete(self):
+        # Select primaries after usual catchup (not view change)
+        ledger = self.node.getLedger(AUDIT_LEDGER_ID)
+        if len(ledger) == 0:
+            if self.viewNo != 0:
```

Comment:
As was mentioned in previous round of review, please remove this condition and error since this is a normal situation.